irap-fedora/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM nunofonseca/irap_fedora:v1.0.3
+
+# Install time
+RUN yum -y update && yum -y install time
+
+# Add libtbb.so.2 and the appropriate libm.so.6 to LD_LIBRARY_PATH
+ENV LD_LIBRARY_PATH=/opt/irap/bin/salmon/lib/libtbb.so.2:/usr/lib64
```

```suggestion
ENV LD_LIBRARY_PATH=/opt/irap/bin/salmon/lib:/usr/lib64:$LD_LIBRARY_PATH
```
unless that before it was just /usr/lib64 only. Still you probably want to cut shot to salmon/lib.

---

This is the first occurrence of LD_LIBRARY_PATH in the container.  There is no LD_LIBRARY_PATH set in the container until the sourcing of the script that sets up the irap environment (which expects and appends LD_LIBRARY_PATH), so I didn't include `:$LD_LIBRARY_PATH` in this line.  I've also added /usr/lib64 so the container can use the correct `libm.so.6`.  This is to resolve a complaint from bedtools stemming from the use of the incompatible `/opt/irap/bin/salmon/liblibm.so.6` (also the reason why I specified the salmon/lib/libtbb.so.2, but the change in preference in 7988421a0ff55ba5dfe001f8d3d2ec7b75de358e resolves that cleanly).